Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-plp-images/index.jsx

```diff
@@ -7,10 +7,10 @@ import { updateTrackingParam } from '../../utils/imageSrcset'
 const CloudinaryPlpImage = ({ cloudinaryImage = {}, dynamicImageProps = {}, image = {} }) => {
     if (typeof window !== 'undefined' && cloudinaryImage.url) {
         useEffect(() => {
-            const width = cloudinaryImage.c_autoResponsiveDimensions.replace(
+            const width = cloudinaryImage.c_autoResponsiveDimensions.replace('w_auto,c_scale', 'w_auto,c_limit').replace(
                 'auto',
```

Comment:
@asad-rafter this additional replace doesn't make any sense to me. It hard-code the React backend screen width?

Let's remove it